README.md

```diff
@@ -345,6 +345,23 @@ would give the following resolutions:
 |b        |/scope2/scope3/foo.mjs |/b-3.mjs      |
 |c        |/scope2/scope3/foo.mjs |/c-1.mjs      |
 
+#### `<base>` element
+
+When `<base>` element is present in the document, all URLs and URL-like specifiers in the import map are converted to absolute URLs using the `href` from `<base>`.
+
+The right-hand side of the mapping should be absolute URLs starting with `./`. "Bare" relative URLs like `node_modules/moment/src/moment.js` will not work for reasons stated in [[Bare specifiers for JavaScript modules]](#bare-specifiers-for-javascript-modules).
+
+```html
+<base href="https://www.unpkg.com/vue/dist/">
+<script type="importmap">
+{
+  "imports": {
+    "vue": "./vue.runtime.esm.js",
+    // resolves to https://www.unpkg.com/vue/dist/vue.runtime.esm.js
```

Comment:
Since comments are not valid JSON, I think this is a bit confusing for people. Maybe insert something like

```html
<script>
import("vue"); // resolves to https://www.unpkg.com/vue/dist/vue.runtime.esm.js
</script>
```

below?